astro/src/content/docs/release-notes/index.mdx

```diff
@@ -13,17 +13,119 @@ import ReleaseNoteHeading from 'src/components/docs/release-notes/ReleaseNoteHea
+  * Resolves [GitHub Issue #2778](https://github.com/FusionAuth/fusionauth-issues/issues/2778)
+
+* The preview window for simple themes in the admin UI was showing un-styled pages when using the Firefox browser. Sorry Firefox users, we ask for your forgiveness. 😔
+  * Resolves [GitHub Issue #2794](https://github.com/FusionAuth/fusionauth-issues/issues/2794)
+
+* The default `orderBy` parameter value for the [Group Member Search API](/docs/apis/groups#search-for-group-members) did not provide a consistent ordering of results because the default sort was on `insertInstant ASC` which may not always be unique. This API is used by the SCIM Groups Resource API which then can cause inconsistent results for the SCIM client. The default `orderBy` is now set to `insertInstant ASC, userId ASC, groupId ASC` to ensure a consistent result between API calls.
+  * Resolves [GitHub Issue #2798](https://github.com/FusionAuth/fusionauth-issues/issues/2798)
+
+* When using the simple theme editor in the admin UI, the color picker did not always render next to the input field. The color picker will now always correctly render adjacent to the input field you select.
+  * Resolves [GitHub Issue #2803](https://github.com/FusionAuth/fusionauth-issues/issues/2803)
+
+* Newlines and tabs were not being rendered when viewing audit entries in the view dialog from the admin UI. If you are using new lines or tabs in your audit log messages, you may not enjoy viewing them in all of their their intended glory!
+  * Resolves [GitHub Issue #2808](https://github.com/FusionAuth/fusionauth-issues/issues/2808)
+
+* When using the interactive maintenance mode to upgrade your database schema, it is possible that you had to click the Submit button twice to exit maintenance mode. This was only a cosmetic issue but may be annoying of confusing to the user. We are sorry if you had to click the submit button twice. 😬
+  * Resolves [GitHub Issue #2815](https://github.com/FusionAuth/fusionauth-issues/issues/2815)
+
+### Enhancements
+* Add the new health check endpoint (`/api/health`) that was added in `1.51.1` to the client libraries.
+  * Resolves [GitHub Issue #2804](https://github.com/FusionAuth/fusionauth-issues/issues/2804)
+
+### Internal
+* For users in FusionAuth Cloud, attempting to save a Simple theme may result in an error.
+  * Resolves [GitHub Issue #2777](https://github.com/FusionAuth/fusionauth-issues/issues/2777)
+
+* An equals (`=`) sign in query parameter value was not being parsed correctly. There are no known issues related to this bug as generally speaking an equals (`=`) sign will be URL encoded as `%3D`. However because it is legal use an equals (`=`) sign un-encoded in a query string name or query string value, this has been corrected.
```

Comment:
```suggestion
* An equals (`=`) sign in query parameter value was not being parsed correctly. There are no known issues related to this bug as generally speaking an equals (`=`) sign will be URL encoded as `%3D`. However, because it is legal use an equals (`=`) sign un-encoded in a query string name or query string value, this has been corrected.
```